### ISSUE TITLE
CMake load GCC to fix linker issue - Stanage R9

### ISSUE THREAD

**Issue by Carldkennedy** [rcgsheffield/sheffield_hpc#2132]
Add note that if using CMake to compile then need to module load GCC/10.2 GCC/10.3 for those versions. Since new EL9 linker intentionally doesn't link  <= GCCcore/10.3

Example error:
```
usr/bin/ld.gold: /opt/apps/tuos/el9/software/live/GCCcore/10.2.0/lib64/libstdc++.so.6: version `GLIBCXX_3.4.29' not found (required by /usr/bin/ld.gold)
```

Fix:
```
if [[ "$version" == *GCCcore-10.2.0* ]]; then
    module load GCC/10.2.0
elif [[ "$version" == *GCCcore-10.3.0* ]]; then
    module load GCC/10.3.0
fi
```
https://uos-it-services.atlassian.net/browse/R9ST-79

**Comment by jkwmoore** [comment]
This needs more context / explanation?

**Comment by Carldkennedy** [comment]
Closed by #2193
